### ISSUE TITLE
Suggest Turning on Group Functionality for Discourse

### ISSUE THREAD

**Issue by MadeofTin** [ethereum-magicians/scrolls#14]
I did some digging and it is possible to create self selecting groups around rings which seems to be ideal for us. https://meta.discourse.org/t/self-selecting-groups/93646

> When you manage a group, check both of these options
 :white_check_mark: Allow users to join the group freely (Requires publicly visible group)
:white_check_mark: Allow users to leave the group freely

Can we turn on this functionality?

**Comment by bmann** [comment]
It's turned on by default, but group _creation_ isn't open to everyone. I think we'll want to use the same process for Forming a Ring, where a group being created is part of it.

@ligi want to have a Wallet Ring "group" to start with?

@jpitts can you set that up?

**Comment by ligi** [comment]
yes - a wallet group to e.g. easily be able to invite all members to a gathering would be great!

**Comment by jpitts** [comment]
As part of the process for setting up a Forum Group for the Ring, we will need for the Ring leaders (sounds nefarious LOL) to specify the configuration. 

![image](https://user-images.githubusercontent.com/509756/43865924-de2b0080-9b18-11e8-9cd1-4b0c6c92aecd.png)

**Comment by jpitts** [comment]
For now, I will create a demo Forum Group for a new membership-only ring I am proposing called "Magician Operations Ring". This will be a Ring for those involved in managing resources such as the Forum and GitHub repo.

**Comment by MadeofTin** [comment]
Would each ring need is own lead? or is there a group of leads that have access to these Permissions? In the formation requirements of a ring we can stipulate that a Ring Lead be identified to handle these roles.

Also, under the Operations ring would @bmann 's work with the Events site fit?
I can see an Events Ring that focus on bettering events in Ethereum overall. Or, I could see an Operations Ring be over the management of maintaining the repo and identifying development needs as they come up. Thoughts Bmann ?

**Comment by jpitts** [comment]
@MadeofTin:

We'd need one or more leads for a Ring in order to grant the group privileges, ring formation is a good time to do this, and at each council we can make it part of the process to update the status of each Ring.

Operations is definitely for managing resources such as the events website, the repo, infrastructure of the Forum, dev needs, etc. 

There is a moderators role in the Forum, perhaps for the time being that can be Operations... but that role could eventually go under a group concerned with the Magicians' community.
